src/lib/curl/base.h

```diff
@@ -32,6 +32,16 @@ extern "C" {
 #include <curl/curl.h>
 #include <freeradius-devel/server/request.h>
 #include <freeradius-devel/util/event.h>
+#include <freeradius-devel/server/module.h>
+
+#define FR_CURL_SET_OPTION(_x, _y)\
+do {\
+	if ((ret = curl_easy_setopt(randle->candle, _x, _y)) != CURLE_OK) {\
```

Comment:
Add a locally scoped `int _ret` var too and use it in the assignment and log message.